File: frontend/lib/models/carro.dart
class Carro {
  int id;
  String nome;
  String marca;
  String modelo;

  Carro(this.id, this.nome, this.marca, this.modelo) {
    this.id = id;
    this.nome = nome;
    this.marca = marca;
    this.modelo = modelo;
  }

  Carro.fromJson(Map json)
    : id = json['id'],
      nome = json['nome'],
      marca = json['marca'],
      modelo = json['modelo'];

}

File: backend/src/entity/Carro.ts
import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm'

@Entity()
export class Carro {

    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    nome: String;

    @Column()
    marca: string;

    @Column()
    modelo: string;

    @Column({default: false})
    finished: boolean;

    @CreateDateColumn()
    created_at: Date;

    @UpdateDateColumn()
    updated_at: Date;

}